docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -299,6 +299,294 @@ When I scan barcode from context and save result to scenario variable `qrCodeLin
 Then `${qrCodeLink}` is equal to `https://www.example.com`
 ----
 
+=== Element Steps
+
+=== Upload file
+
+This step uploads a file with the given relative path.
+
+NOTE: A relative path starts from some given working directory, avoiding the need to provide the full absolute path (i.e. 'about.jpeg' for the files placed in the src/main/resources directory or '/story/uploadfiles/about.png')
+
+[source,gherkin]
+----
+When I select element located by `$locator` and upload `$resourceNameOrFilePath`
+----
+
+_Deprecated syntax (will be removed in VIVIDUS 0.7.0)_:
+[source,gherkin]
+----
+When I select element located `$locator` and upload file `$filePath`
+----
+
+* `$locator` - The <<_locator,locator>> to find an element.
```

Comment:
we need to mention it's required to specify locator for `input` element with attribute `type=file`